doc/semitrans.xml

```diff
@@ -349,3 +349,29 @@ gap> FixedPointsOfTransformationSemigroup(S);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="WreathProduct">
+  <ManSection>
+    <Oper Name = "WreathProduct" Arg = "S, G"/>
```

Comment:
We should also have the version where `Arg = "S, G"`.  You can just add the line right below this.